docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -547,7 +547,7 @@ def test_uses_resource() -> None:
     assert result == {"foo": "bar"}
 ```
 
-If you use a context object in your function, you can use <PyObject object="build_op_context" /> to generate the `context` object, because under the hood the function decorated by `@asset` is an op.
+If you use a context object in your function, you can use <PyObject object="build_asset_context" /> to generate the `context` object, because under the hood the function decorated by `@asset` is an op.
```

Comment:
Do we need to keep the `under the hood this is an op` here? As we're no longer mentioning op context, it doesn't seem like it's needed

Reply:
ah yes 100% i think i just glazed over what this sentence actually said. thanks for catching!